rubicon_ml/viz/dataframe_plot.py

```diff
@@ -89,8 +89,15 @@ def load_experiment_data(self):
         `dataframe_name` must have the same schema.
         """
         self.data_df = None
+        count = 0
 
         for experiment in self.experiments:
+            if len(experiment.dataframes()) == 0:
+                print(
```

Comment:
import warnings and use `warnings.warn` instead. then you won't need the "WARNING: " text either in the string output